### ISSUE TITLE
[Bug report] - Incorrect identification on Desktop: session type is 'mobile'

### ISSUE THREAD

**Issue by ittat** [capyloon/nutria#60]
**Describe the bug**

I found that Nutria can not wake up on MacOS because of  `embedder.sessionType` get value is  "mobile".  

console log:  
```js 
console.log: "-*- Shell.js: New web embedder created."
console.log: "ext-Embedding: setDelegate"
console.log: "WebExtensionsDelegate: setProvider"
console.log: "Embedding: === Layer Manager: WebRender for chrome://system/content/index.html"
console.log: "Embedding: WebExtension Signature needed? false"
console.log: "Embedding: session type is 'mobile'"
console.log: "Timing 'DOMContentLoaded, embedder is [object Promise]' at 5ms"
console.log: "embedderSetupDone, embedder is [object EventTarget]"
```
The conflicting code is located here:  
https://github.com/capyloon/nutria/blob/main/apps/system/js/power_manager.js#L128

** Environment **
- Desktop Mac

**To Reproduce**
null

**Expected behavior**
Is embedder.sessionType  "desktop" ?

**Screenshots**
null. 

**Additional context**
null

**Comment by fabricedesre** [comment]
If you want to run in with a desktop session, you need to launch with `./jackady dev --type desktop`. But that will also disable the virtual keyboard.

As a workaround when the session is `mobile`, the screen should not lock if the laptop is plugged in. On Linux, refocusing the whole Capyloon window usually makes it possible to unlock with the swipe up gesture. I need to check if that also works on Mac...

In any case we need a proper fix for the `mobile` sessions running on laptops, this is quite annoying indeed.

**Comment by ittat** [comment]
@fabricedesre thanks,  it is working well with`./jackady dev --type desktop`.  When the session is `mobile`, the system turns off the screen when idle.   https://github.com/capyloon/nutria/blob/main/apps/system/js/power_manager.js#L232

**Comment by fabricedesre** [comment]
Yes, that's expected to turn off the screen when idle on mobile. But sometimes it gets hard to unlock when using the mobile profile on desktop/laptop (this works fine on devices).
